packages/vue/src/components/SpAcc/SpAcc.vue

```diff
@@ -0,0 +1,289 @@
+				<div v-else-if="state.modalPage === 'error'">
+					<SpWarningIcon style="margin-bottom: 20px" />
+					<h3>Keplr cannot launch</h3>
+				</div>
+			</template>
+			<template v-slot:body>
+				<div style="max-width: 320px; text-align: center; margin: auto">
+					<div v-if="state.modalPage === 'connect'">
+						<p v-if="isKeplrAvailbe">
+							Connect your Keplr wallet via the Keplr browser extension to use
+							this app.
+						</p>
+						<p v-else>
+							Install & connect your Keplr wallet via the Keplr browser
+							extension to use this app.
+						</p>
+					</div>
+					<div v-else-if="state.modalPage === 'connecting'">
+						<div style="margin-top: 2rem">
+							<SpSpinner />
+						</div>
+						<SpButton
+							aria-label="Cancel"
+							type="secondary"
+							@click="state.modalPage = 'connect'"
+							style="margin-top: 3rem"
+						>
+							Cancel
+						</SpButton>
+						<div class="external-link" style="margin-top: 2rem">
+							Having trouble opening Keplr?
+						</div>
+					</div>
+					<div v-else-if="state.modalPage === 'error'" style="padding: 20px 0">
+						<div class="external-link">
+							<span>Keplr troubleshooting</span>
+							<SpExternalArrowIcon style="margin-left: 0.5rem" />
+						</div>
+					</div>
+				</div>
+			</template>
+			<template v-if="isKeplrAvailbe" v-slot:footer>
+				<div v-if="state.modalPage === 'connect'">
+					<SpButton
+						aria-label="Connect Keplr"
+						type="primary"
+						@click="tryToConnectToKeplr"
+					>
+						Connect Keplr
+					</SpButton>
+				</div>
+				<div
+					v-if="state.modalPage === 'error'"
+					style="gap: 10px; display: flex; justify-content: center"
+				>
+					<SpButton
+						aria-label="Connect Keplr"
+						type="secondary"
+						@click="state.connectWalletModal = false"
+					>
+						Cancel
+					</SpButton>
+					<SpButton
+						aria-label="Connect Keplr"
+						type="primary"
+						@click="state.modalPage = 'connect'"
+					>
+						Try again
+					</SpButton>
+				</div>
+			</template>
+		</SpModal>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, computed, ComputedRef } from 'vue'
+import { useStore } from 'vuex'
+
+import SpModal from '../SpModal'
+import SpButton from '../SpButton'
+import SpAccountDropdown from '../SpAccountDropdown'
+import SpSpinner from '../SpSpinner'
+import SpProfileIcon from '../SpProfileIcon'
+import SpKeplrIcon from '../SpKeplrIcon'
+import SpWarningIcon from '../SpWarningIcon'
+import SpExternalArrowIcon from '../SpExternalArrow'
+import SpChevronDownIcon from '../SpChevronDown'
+
+import useKeplr from '@/composables/useKeplr'
+import { Wallet } from '@/utils/interfaces'
+
+export default defineComponent({
+	name: 'SpAcc',
+
+	components: {
+		SpModal,
+		SpButton,
+		SpSpinner,
+		SpProfileIcon,
+		SpAccountDropdown,
+		SpKeplrIcon,
+		SpWarningIcon,
+		SpExternalArrowIcon,
+		SpChevronDownIcon
+	},
+
+	setup() {
+		// state
+		let state = reactive({
```

Comment:
Let's type state as in [SpTx](https://github.com/tendermint/vue/blob/7960205087d62318209160dd88fae4cd2c580def/packages/vue/src/components/SpTx/SpTx.vue#L308) please!